opentelemetry-user-events-metrics/examples/basic.rs

```diff
@@ -32,13 +32,42 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         Some("test_url"),
         Some(vec![KeyValue::new("key", "value")]),
     );
-    let c = meter
+    // Create a Counter Instrument.
+    let counter = meter
         .f64_counter("counter_test")
         .with_description("test_decription")
         .with_unit(Unit::new("test_unit"))
         .init();
+    // Create a UpCounter Instrument.
+    let updown_counter = meter.i64_up_down_counter("my_updown_counter").init();
 
-    c.add(
+    // Create a Histogram Instrument.
+    let histogram = meter
+        .f64_histogram("my_histogram")
+        .with_description("My histogram example description")
+        .init();
+
+    // Create a ObservableCounter instrument and register a callback that reports the measurement.
+    let gauge = meter
```

Comment:
can you add all instruments ? Observablecounter, ObservableUpDownCounter are missing.
And also, it'll be nice to emit metric in a infinite loop with a 1 sec sleep - its a lot easier to test, if metrics are flowing continuously.